### ISSUE TITLE
hledger bs output formatting issue

### ISSUE THREAD

**Issue by markokocic** [simonmichael/hledger#534]
hledger `bs` formats the output differently than `bal`, e.g. it is not compressing the output by default in case of single subaccount.

Example in http://hledger.org/manual.html#balancesheet shows `bs` to compress the subaccounts.

```
marko@sky:~$ cat test.journal
2017/01/01 Test
        expenses:exp1  200
        assets:bank1:current

marko@sky:~$ h bal -f test.journal
                -200  assets:bank1:current
                 200  expenses:exp1
--------------------
                   0
marko@sky:~$ h bs -f test.journal
Balance Sheet

Assets:
                -200  assets
                -200    bank1
                -200      current
--------------------
                -200

Liabilities:
--------------------
                   0

Total:
--------------------
                -200
marko@sky:~$
```

**Comment by simonmichael** [comment]
Good catch. That eliding feature is costly, I almost wish we scrapped it. It works with cashflow:
```
~/src/hledger$ hledger-1.1 -f examples/sample.journal cf
Cashflow Statement

Cash flows:
                 $-1  assets
                  $1    bank:saving
                 $-2    cash
...
~/src/hledger$ hledger -f examples/sample.journal cf
Cashflow Statement

Cash flows:
                 $-1  assets
                  $1    bank:saving
                 $-2    cash
...
```

but not with balancesheet:
```
~/src/hledger$ hledger-1.1 -f examples/sample.journal bs
Balance Sheet

Assets:
                 $-1  assets
                  $1    bank:saving
                 $-2    cash
...
~/src/hledger$ hledger -f examples/sample.journal bs
Balance Sheet

Assets:
                 $-1  assets
                  $1    bank
                  $1      saving
                 $-2    cash
...
```
Also as previously noted balancesheet in single column mode ignores --flat, possibly related.

**Comment by mstksg** [comment]
After some digging, it looks like it's an issue with `bal --historical` --

~~~
$ hledger -f test.journal bal --historical
                -200  assets
                -200    bank1
                -200      current
                 200  expenses
                 200    exp1
--------------------
                   0
~~~

So once `bs` was written to be a wrapper over `bal ^assets ^liabitilies --historical`, this popped up.

The reason why `cashflow` displays the correct behavior; it's just a straight `bal --change`, which properly elides.

**Comment by simonmichael** [comment]
Aha, interesting.

Do people like the eliding behaviour ?

**Comment by markokocic** [comment]
Yes. Less verbose output with less repetition.

Simon Michael <notifications@github.com> schrieb am Fr. 31. März 2017 um
21:12:

**Comment by Xitian9** [comment]
I believe this was fixed by #1256 and #1269, and this can be closed.
